modules/javafx.controls/src/test/java/test/javafx/scene/control/DialogPaneTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You need to revert this change. There must be only a single year listed if the start year and last modified year are the same.